gui/GameConqueror.py

```diff
@@ -905,7 +907,7 @@ def update_scan_result(self):
             # temporarily disable model for scanresult_liststore for the sake of performance
             self.scanresult_liststore.clear()
             for line in lines:
-                line = str(line)
+                line = str(line.decode("utf-8"))
```

Comment:
Maybe the auto-conversion to str() is only valid for Python 3. At the other location I had to add str() for Python 2.